CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added the callback fired when the render button is clicked to `PopOver` UI component
```

Comment:
Just a thought, should we reword this log? For example:
Added an optional prop `onRenderButtonClick` to pass a callback function to the `PopOver` UI component. This callback will be called when the `PopOver` UI component is clicked.